clients/src/main/java/org/apache/kafka/common/requests/TxnOffsetCommitResponse.java

```diff
@@ -47,6 +47,9 @@
  *   - {@link Errors#UNKNOWN_MEMBER_ID}
  *   - {@link Errors#FENCED_INSTANCE_ID}
  *   - {@link Errors#ILLEGAL_GENERATION}
+ *   - {@link Errors#INVALID_TXN_STATE}
```

Comment:
There are some issues with these new errors. (Invalid_producer_id_mapping is fine and abortable). The other two added errors are currently fatal, but I think it would be better if they were abortable. 

Let me know what you think.

Reply:
I think we can make concurrent transactions retriable by converting it to coordinator loading before sending the response.

As for the invalid exceptions, we can convert to one of the abortable errors like Unknown member ID. 

In part 2, we can fix this to be better errors that the client can understand better.

Reply:
Silly me -- we already convert the retriable ones now :) ReplicaManager converts to Not enough replicas which converts back to coordinator load in progress. I made the other two errors unkonwn member id.